AirCasting/Utils/UserState.swift

```diff
@@ -4,5 +4,13 @@
 import Foundation
 
 class UserState: ObservableObject {
+    enum State {
+        case loggingOut
+        case deleting
+        case none
```

Comment:
I like that you used an `enum` here instead of going forward with multiple `Bool` flags 🚀

However I don't quite like the `none` user state. I know its purpose - its anything but `loggingOut` and `deleting`. Maybe `other`?